### DISCUSSION TITLE
android app crash when i control it

### DISCUSSION THREAD

**Discussion by 385628424** [rustdesk/rustdesk#8953] [General]
### Bug Description

I use mi11 to remotely control honor10, but every time my mi11 starts to control it, rustdesk in honor10 will automatically crash and exit as soon as I start the operation or operate it for a while.

### How to Reproduce

use honor or huawei phone to controled

### Expected Behavior

normal contral

### Operating system(s) on local side and remote side

android ->android

### RustDesk Version(s) on local side and remote side

1.2.6 -> 1.2.7

### Screenshots

never

**Comment by rustdesk** [comment]
Can you try turn off this on controlled android?

![image](https://github.com/user-attachments/assets/43e304fd-676f-4ffa-a45c-16fa86f28000)
